proposals/NNNN-compilation-flags-with-values.md

```diff
@@ -0,0 +1,63 @@
+# Accept Values for Custom Compilation Flags
+
+* Proposal: [SE-NNNN](NNNN-compilation-flags-with-values.md)
+* Authors: [John Holdsworth](https://github.com/johnno1962)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+
+*During the review process, add the following fields as needed:*
+
+* Implementation: [apple/swift#19585](https://github.com/apple/swift/pull/19585)
+* Toolchain: [available here](http://johnholdsworth.com/swift-LOCAL-2018-09-26-a-osx.tar.gz)
```

Comment:
- Use [`@swift-ci build toolchain`](https://github.com/apple/swift/blob/master/docs/ContinuousIntegration.md#build-swift-toolchain) for macOS and Linux?
- Remove `*During the review process*` and empty lines?
- Remove unused "Decision Notes" field?